lib/compiler.js

```diff
@@ -263,6 +265,36 @@ function sortByInheritance(instructions) {
     });
 }
 
+function verifyModelDefinitions(rootDir, modelDefinitions) {
+  if (!modelDefinitions || modelDefinitions.length < 1) {
+    return undefined;
+  }
+
+  var registry = {};
+  modelDefinitions.forEach(function(definition, idx) {
+    try {
+      var fullPath = path.resolve(rootDir, definition.sourceFile);
+      definition.sourceFile = require.resolve(fullPath);
+    } catch (err) {
+      debug('Model source code not found: %s - %s', definition.sourceFile, err.code || err);
+      definition.sourceFile = undefined;
+    }
```

Comment:
@bajtos, added.
unit test is not needed, it is tested in "should loads model without definition".

Since the block is in try/catch it will not load it anyway, just waste some resources try to get it... This is the implementation in "loadModelDefinition()"

I blocked it with an if statement just in case...